ooiui/templates/science/plotting.html

```diff
@@ -530,16 +537,15 @@
       }
     });
 
-    this.listenTo(this, 'AnnotationTableView:onAddClick', function() {
-
-      var bounds = self.views.timeseriesView.views.highchartsView.chart.xAxis[0].getExtremes()
-
-      var min = moment.utc(bounds.userMin).format("YYYY-MM-DDTHH:mm:ss.000")+"Z";
-      var max = moment.utc(bounds.userMax).format("YYYY-MM-DDTHH:mm:ss.000")+"Z";
+    this.listenTo(this, 'AnnotationTableView:onAddClick', function() {      
+      var min = moment.utc(self.views.svgPlotControlView.model.get('start')).format("DD/MM/YYYY HH:mm");
```

Comment:
@birdage Why are we changing the time format from what uFrame accepts?  Does it matter?